trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class TrimmerDecorator < Decorator
+  MAX_LENGTH = 10
+
+  def correct_name
+    super.correct_name[0, MAX_LENGTH]
+  end
+end
```

Comment:
- [ ] The name has a maximum of 10 characters. If it's longer it should trim the word but it return 11 character [0,10] means 11 characters. You can rewrite you code as :
``@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name``